FluentPDO/BaseQuery.php

```diff
@@ -114,9 +117,15 @@ public function execute() {
 		}
 
 		$time = microtime(true);
-		if ($result && $result->execute($parameters)) {
+		if ($result && @$result->execute($parameters)) {
```

Comment:
Please don't suppress errors with @. There is almost never any valid reason for this